Source_Code/Server/ClueLessServer.java

```diff
@@ -458,6 +458,12 @@ else if ( ! ( possibleRefutations.getWeapons() == null ) )
                     sendToAllPlayers(new SuggestionPassed( p.PlayerName ));
                 }
             }
+            // If it comes back to us, do we lie or do nothing? I think do nothing is better?
```

Comment:
Do nothing.
If I only need to figure out what weapon was used, I can use a character I have in my hand to not give any info away, or to make sure someone can't show me a character when I want to see if they have a weapon.